src/components/organisms/seriesDetailEpisodes/SeriesDetailEpisodes.tsx

```diff
@@ -34,6 +40,7 @@ export function SeriesDetailEpisodes({
 
   const seeAllEpisodesButton = useMemo(() => {
     if (shouldSeeAllEpisodes) return null
+    if (seriesInfo.orderedVideoTitles.length <= 3) return null
```

Comment:
Don't show the "See more episodes" button if there are only three or fewer episodes to show